docs/src/pages/[platform]/getting-started/installation/index.page.mdx

```diff
@@ -27,6 +27,10 @@ export async function getStaticProps() {
   {({ platform }) => import(`./dependencies.${platform}.mdx`)}
 </Fragment>
 
+<Fragment platforms={['web']} useCommonWebContent>
+  {({ platform }) => import(`./shims.${platform}.mdx`)}
+</Fragment>
+
```

Comment:
Found a docs bug:

![image](https://github.com/aws-amplify/amplify-ui/assets/43682783/202d8f9f-2939-44d1-9210-7312fb0cc322)

It tries to load `shim.<platform>.mdx` for all frameworks. Can we fix this and scope this section only to Angular?

Reply:
Added fix to update `shims.angular.mdx` and remove `shims.web.mdx`